File: src/net/eduard/chat/command/ChatReloadCommand.java

package net.eduard.chat.command;

import net.eduard.chat.EduChat;
import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;

import net.eduard.api.lib.manager.CommandManager;

public class ChatReloadCommand extends CommandManager {

	
	public ChatReloadCommand() {
		super("chatreload", "educhatreload");
	}
	@Override
	public boolean onCommand(CommandSender sender, Command command,
			String label, String[] args) {
		EduChat.getInstance().reload();
		sender.sendMessage("§aSistema de chat recarregado");
		return true;
	}
}

File: src/main/java/net/eduard/chat/command/ChatReloadCommand.kt
package net.eduard.chat.command

import net.eduard.api.lib.manager.CommandManager
import net.eduard.chat.EduChatPlugin
import org.bukkit.command.CommandSender

class ChatReloadCommand : CommandManager("chatreload", "educhatreload") {
    init {
        description = "Executa o reload do plugin"
    }

    override fun command(sender: CommandSender, args: Array<String>) {
        EduChatPlugin.instance.reload()
        sender.sendMessage("§aSistema de chat recarregado")
    }

}